src/components/Header/IconLink.jsx

```diff
@@ -0,0 +1,30 @@
+import { IconButton } from "@material-ui/core";
+
+import Tooltip from '@material-ui/core/Tooltip';
+import FavoriteIcon from "@material-ui/icons/Favorite";
+import AddShoppingCartIcon from '@material-ui/icons/AddShoppingCart';
+import FaceIcon from '@material-ui/icons/Face';
```

Comment:
Hello! Pienso que esta librería podría sustituirse por componentes creados por nosotros (por ejemplo el tooltip podría ser un `title`)

Y sobre los iconos, podríamos usar esta librería que pesa menos que material! https://fontawesome.com/how-to-use/on-the-web/using-with/react

¿Como lo veis? @elisasoto

Reply:
Super bien, he creado el componente para ser reutilizado en muchos lugares i.e: el search input, el footer con logos de redes sociales etc... solo habría que agregar los logos en el objeto de icons y voilá 🙉